### ISSUE TITLE
[Suggestion] Why not make super circuit maker open source?

### ISSUE THREAD

**Issue by jambonmcyeah** [amadornes/SCM-Localization-Issues#305]

**Comment by Soundtoxin** [comment]
Yeah, why not? Just because Minecraft doesn't respect your freedom doesn't mean mods for it shouldn't either!

**Comment by dokranke** [comment]
Because SCM is just a very small part of a mod that amadornes is making. Also he doesn't want that anyone works with the api yet, because it could maybe get changed very much

**Comment by jambonmcyeah** [comment]
@gDKdev Are you allowed to tell me what he is making?

**Comment by dokranke** [comment]
He is posting it on twitter, so it's not a secret

**Comment by meponderR** [comment]
He made it open source for 1.13.2. [Click Here](https://github.com/amadornes/SuperCircuitMaker)

**Comment by Soundtoxin** [comment]
I don't see a LICENSE file. Having it on GitHub doesn't mean it's "open source".
